dashboard/app/assets/stylesheets/application.scss

```diff
@@ -855,6 +855,11 @@ body.embedded_iframe {
 $default-modal-width: 640px;
 
 .modal {
+  @media screen and (max-height: 600px) {
+    max-height: 80vh;
+    overflow-y: scroll;
```

Comment:
Sometimes we use `auto` so that scrollbars aren't shown if they aren't needed.

Reply:
yeah, thanks for noticing. For some reason I was thinking this won't work, but I double checked now and it works correctly as well.